### ISSUE TITLE
ELPA automation scripts generate a duplicate of shell-command-plus

### ISSUE THREAD

**Issue by AndersonTorres** [NixOS/nixpkgs#119586]
**Describe the bug**

When I ran the update script for updating ELPA packages, it generated a perfect duplication of ` shell-command-plus`. So I needed to patch the resulting file:

https://github.com/NixOS/nixpkgs/pull/119583

**To Reproduce**

1. `./update-elpa`
2. read file `./elpa-generated.nix`

**Expected behavior**

Every expression should appear once and only once.

**Screenshots**

https://gist.github.com/GrahamcOfBorg/e9bb441968b1a788b81125da3db0f8c1

**Metadata**

 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.100, NixOS, 20.09.20210308.f26c6dd (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4pre20210326_dd77f71`
 - channels(root): `"nixos-21.03pre243434.e0759a49733, nixos-20.03-20.03.2964.252bfe01075, nixos-unstable-21.03pre243434.e0759a49733"`
 - channels(anderson): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
